### DISCUSSION TITLE
Viewing documentation for SQL functions

### DISCUSSION THREAD

**Discussion by sryze** [dbeaver/dbeaver#19259] [Ideas]
I don't know if this was already brought up, but I think it would be useful to be able to click on a built-in function, e.g. `replace()` in PostgreSQL, and quickly view the documentation for it - either as a tooltip or by opening the corresponding web page.

**Comment by mkgrgis** [comment]
Also PostGIS functions can be affected.

**Comment by LonwoLonwo** [comment]
We have this option for the MySQL database because MySQL has a helpful view for it: `mysql.help_topic`
If you know analogs for PostgreSQL - please share. Because another option - keep all this info inside DBeaver code.

**Reply by mkgrgis** [reply]
Unfortunately PostgreSQL or PostGIS haven't structured documentation in a table... Only

- https://www.postgresql.jp/document/current/index.html
- https://docs.postgresql.fr
- https://www.postgrespro.ru/docs/postgresql/15/

and some partially versions

- http://www.postgres.cn/docs/14/index.html
- https://docs.postgresql.tw
- https://www.ibiblio.org/pub/Linux/docs/LuCaS/Postgresql-es/web/navegable/user/user.htm
- https://www.lawebdelprogramador.com/cursos/PostgreSQL/2249-Manual-del-usuario-de-PostgreSQL.html
- https://studylib.es/doc/6161579/manual-del-usuario-de-postgresql
